ra/registration-authority_test.go

```diff
@@ -317,8 +317,8 @@ func TestValidateEmail(t *testing.T) {
 	}{
 		{"an email`", unparseableEmailDetail},
 		{"a@always.invalid", emptyDNSResponseDetail},
-		{"a@always.timeout", "DNS query timed out"},
-		{"a@always.error", "DNS networking error"},
+		{"a@always.timeout", "DNS query timed out during A-record lookup of always.timeout"},
```

Comment:
If the mock resolver's MX query had errored, we would be getting MX record errors. It's not, so it goes on to the A record that does.